src/util/typeAssertions.spec.ts

```diff
@@ -0,0 +1,107 @@
+import {
+  isTextObject,
+  isSerializableFiller,
+  isPattern,
+  isPath,
+  isFiller,
+} from './typeAssertions';
+import { FabricText } from '../shapes/Text/Text';
+import { IText } from '../shapes/IText/IText';
+import { Textbox } from '../shapes/Textbox';
+import { Path } from '../shapes/Path';
+import { Pattern } from '../Pattern/Pattern';
+import { Gradient } from '../gradient/Gradient';
+import { Shadow } from '../Shadow';
+
+describe('typeAssertions', () => {
+  describe('isTextObject', () => {
+    test('can detect FabricText', () => {
+      const text = new FabricText('hello');
+      expect(isTextObject(text)).toBe(true);
+    });
+    test('can detect IText', () => {
+      const text = new IText('hello');
+      expect(isTextObject(text)).toBe(true);
+    });
+    test('can detect Textbox', () => {
+      const text = new Textbox('hello');
+      expect(isTextObject(text)).toBe(true);
+    });
+    test('can detect other subclasses', () => {
+      class NewText extends Textbox {
+        status = 'new';
+      }
```

Comment:
to make it more complete you should add a 
```
static type = 'NextText'
```

Reply:
oh rigth